src/app/space/add-app-overlay/add-app-overlay.component.ts

```diff
@@ -85,5 +96,6 @@ export class AddAppOverlayComponent implements OnDestroy, OnInit {
     this.projectName = this.projectName.toLowerCase();
     this.isProjectNameValid =
       this.dependencyCheckService.validateProjectName(this.projectName);
+    this.isProjectNameAvailable = this.applications.indexOf(this.projectName) === -1 ? true : false;
```

Comment:
Curious if this will be a performance hit? There could be a lot of application names to index.

If the application names are system wide, then this may be better as a REST call? That is, instead of calling the deploymentApiService to get all existing application names.

Reply:
Ok, just wanted to make sure it's limited to a space. Thanks!